core/src/main/kotlin/org/evomaster/core/output/service/PartialOracles.kt

```diff
@@ -0,0 +1,64 @@
+package org.evomaster.core.output.service
+
+import com.google.gson.Gson
+import org.evomaster.core.output.Lines
+import org.evomaster.core.problem.rest.RestCallAction
+import org.evomaster.core.problem.rest.RestCallResult
+import org.evomaster.core.search.gene.ObjectGene
+import org.evomaster.core.search.gene.OptionalGene
+
+
+class PartialOracles {
```

Comment:
provide some comments explaining what are the the partial oracles

Reply:
Will do. I'm expecting this to change quite a bit, but I will do my best to have the explanation there and to keep it as up to date as possible.